### ISSUE TITLE
Error "profile file cannot be null" on IDE restart after the plugin installation

### ISSUE THREAD

**Issue by yaskovdev** [satr/intellij-idea-plugin-connector-for-aws-lambda#4]
```
IntelliJ IDEA 2017.3.4 (Ultimate Edition)
Build #IU-173.4548.28, built on January 30, 2018
JRE: 1.8.0_152-release-1024-b11 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.13.3
```

I installed the plugin and restarted IntelliJ IDEA to apply the changes. During the IDE start the plugin crashed with the following error:

```
profile file cannot be null
java.lang.IllegalArgumentException: profile file cannot be null
	at com.amazonaws.util.ValidationUtils.assertNotNull(ValidationUtils.java:37)
	at com.amazonaws.auth.profile.ProfilesConfigFile.<init>(ProfilesConfigFile.java:142)
	at com.amazonaws.auth.profile.ProfilesConfigFile.<init>(ProfilesConfigFile.java:133)
	at com.amazonaws.auth.profile.ProfilesConfigFile.<init>(ProfilesConfigFile.java:100)
	at io.github.satr.idea.plugin.connector.la.models.ConnectorModel.getCredentialProfiles(ConnectorModel.java:185)
	at io.github.satr.idea.plugin.connector.la.ui.ConnectorPresenterImpl.refreshCredentialProfilesList(ConnectorPresenterImpl.java:130)
	at io.github.satr.idea.plugin.connector.la.ui.ConnectorPresenterImpl.refreshAllLists(ConnectorPresenterImpl.java:155)
	at io.github.satr.idea.plugin.connector.la.ui.ConnectorViewFactory.lambda$runRefreshAllList$23(ConnectorViewFactory.java:250)
	at io.github.satr.idea.plugin.connector.la.ui.ConnectorViewFactory$2.run(ConnectorViewFactory.java:294)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:713)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:157)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:543)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:488)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:94)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:144)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl$2.run(ProgressManagerImpl.java:165)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:315)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by satr** [comment]
Hi,
Thank you your report. Could you tell if you have any profile file existing in the folder as described in this manual?
https://docs.aws.amazon.com/cli/latest/userguide/cli-config-files.html
Particularly files 
~/.aws/credentials
and/or
~/.aws/config
And if exist - if they are populated?

**Comment by satr** [comment]
Fixed in the [version v1.3](https://github.com/satr/intellij-idea-plugin-connector-for-aws-lambda/releases/tag/v1.3)
Thank you.
